product_docs/docs/eprs/7/08_xdb_cli/03_xdb_cli_commands/36_creating_subscription_cli.mdx

```diff
@@ -46,6 +47,64 @@ The file containing the publication server login information of the publication
 
 Comma-separated list of filter IDs identifying the filter rules from the set of available table filters to enable on the corresponding tables in the new subscription. Use the `printpubfilterslist` command to obtain the filter IDs for the available filter rules in the publication (see [Printing a list of filters in a Publication](17_print_publications_filters_list/#print_publications_filters_list)). Don't use white space between the comma and filter IDs.
 
+`customcolmap`
+
+Enter the column mappings between different but compatible columns in the source and target database. Table and schema qualified column names can be used in mappings, or a regular expression can be used to implement a mapping rule on multiple columns. 
+
+!!! Note
+   Column mapping can be added only with the Replication Server CLI; this feature isn't supported by Replication Console. Column mappings are supported only for single-master replication clusters. 
+
+   Column data type mapping supports synchronization between two databases having different but compatible column data types. For example, the data types of a column may be different but data can be populated without any loss. Ensure that there is no possibility of data loss while providing the column data type mapping. This is very important when adding column mapping for columns that act as primary or unique keys as data or precision loss may create inconsistent data across the source and target database.
+
+   Column mapping is currently not supported for large object type columns. 
+!!!
+
+To enter multiple mappings, use a semicolon to separate the column mappings.
+
+Enter mappings in the following format, where `dataType` represents the data type of the column in the target database.
+
+   ```shell
+   SchemaName\.tableName\.columName=dataType
+   ```
+
+The following is an example of custom column mapping: 
+
+   ```shell
+   company\.emp\.emp_id=integer
+   ```
+
+   The `\` characters act as an escape string, while `.` is a reserved character in regular expressions. Use `\.` to represent the `.` character.
+
+The following is an example of regular expression for a column mapping entery: 
+
+   ```shell
+   .*id=integer
+   ```
+
+   Where you map any column whose name ends in `id` to type `integer`. 
+
+`customcolmapfile`
+
+Enter the path of the file containing the column mappings. Multiple custom type mappings can also be provided in the file. You can use either the fully qualify column name or regression expression. Enter each mapping on a new line. 
```

Comment:
The previous comment about the action oriented style of this option description applies here as well.